ui.frontend/src/view/FormFieldBase.js

```diff
@@ -171,13 +171,48 @@ class FormFieldBase extends FormField {
         }
     }
 
+
+    #syncError() {
```

Comment:
Why are we shifting all this to js instead of HTML?

Reply:
This is because to handle the repeatability for aria-describedby, short-description, long-description and error.